docs/source/algorithms/NormaliseToMonitor-v1.rst

```diff
@@ -55,20 +55,26 @@ technically dimensionless.
 Restrictions on the input workspace
 ###################################
 
-The data must be histogram, non-distribution data.
+The data must be histogram, non-distribution data. The exception to this is for workspaces that contain point data with a single
```

Comment:
`The exception` refers only to the "histogram" bit, not the "not-distribution" bit, right?

Reply:
Correct, fixed in the docs.